lib/Peep.rb

```diff
@@ -0,0 +1,14 @@
+require 'pg'
+
+class Peep
+  def self.all
+    if ENV['Environment'] == 'test'
+      connect = PG.connect(dbname: 'chitter_manager_test')
```

Comment:
You stored the value into `connect` but later in the code use `connection`, this would potentially result in a `nil` error on line 8

One possible other way to do it could be (so that it always has a value)
```rb
db_name = ENV['Environment'] == 'test' ? 'chitter_manager_test' : 'chitter_manager'
connection = PG.connect(dbname: db_name)
```

Reply:
This is on the right track towards the fix. The next error message I got was:

```
PG::UndefinedTable:
       ERROR:  relation "comment" does not exist
       LINE 1: SELECT * FROM comment;
```